### DISCUSSION TITLE
Could Pandora be integrated?

### DISCUSSION THREAD

**Discussion by ScottVinzant** [rern/rAudio#427] [Feature Requests]
@rern 
Would it be possible to get Pandora integrated into the interface?
IDK if Pandora has an API.
I have seen "PianoBar", but don't know if that could have some simple controls wrapped around it to make it function as a simple controller.

Thanks!
Scott M. Vinzant

**Comment by skiphansen** [comment]
Is anyone working on this?  If not I might take a whack at it.  My wife loves Pandora.
